core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -168,7 +184,7 @@ class GroupCoordinator(val brokerId: Int,
           addPendingMemberExpiration(group, newMemberId, sessionTimeoutMs)
```

Comment:
nit: could we append the comment above with
`If member id required (dynamic membership)`
just to make it even more clearer than static members won't be pending members (I know this is noted in `doJoinGroup`

Reply:
+1